src/components/targets/modals/deleteTargetModal.tsx

```diff
@@ -16,8 +16,16 @@ const deleteTarget = async (id: number) => {
   }
 }
 
-const DeleteTargetModal = ({setShowModal, areaLength, title, topic, id}: 
-  { setShowModal: React.Dispatch<React.SetStateAction<boolean>>, areaLength: string, title: string, topic: string, id: number }) => {
+const DeleteTargetModal = ({setShowModal, showChat, areaLength, title, topic, id}: 
+  { setShowModal: React.Dispatch<React.SetStateAction<boolean>>, showChat?: (conversationId: number) => void,
+    areaLength: string, title: string, topic: string, id: number
+  }) => {
+
```

Comment:
WDYT? This can me your code a little bit more readable 
```suggestion
interface DeleteTargetModalProps {
// Your types here....
}

const DeleteTargetModal = ({setShowModal, showChat, areaLength, title, topic, id}: DeleteTargetModalProps) => {

```

Reply:
And if you like this way, you can apply it in every component